lib/brains/tests/test_forms.py

```diff
@@ -131,12 +134,32 @@ def test_ok(self):
 
     def test_ok_method(self):
         seller, seller_product = create_seller()
+        self.product_mock.get('moz-brick').recurrence = None
         method = create_method(create_braintree_buyer()[1])
+
         form, errors = self.process({
-            'amount': '5',
+            'amount': '10.00',
             'paymethod': method.get_uri(),
             'product_id': seller_product.public_id
         })
         assert not errors, errors.as_text()
         assert 'payment_method_nonce' not in form.braintree_data
         eq_(form.braintree_data['payment_method_token'], method.provider_id)
+
+    def test_recurring(self):
+        seller, seller_product = create_seller()
+        form, errors = self.process({
+            'amount': '10.00',
+            'nonce': 'nonce',
+            'product_id': seller_product.public_id
+        })
+        eq_(form.errors.as_data()['product_id'][0].code, 'invalid')
+
+    def test_amount(self):
```

Comment:
why is this amount invalid? Just a guess but maybe the test should be named `test_fail_if_amount_too_low` ?